### PR TITLE
Store show

### PR DESCRIPTION
**Pull request by liambarstad**
LIST OF CHANGES:
- Add store page title and item styling
- Add truncated descriptions and titles
- Item elements in line

THINGS I'M UNSURE ON:

RSPEC GREEN?
YES

*MENTION GROUP MEMBER TO REVIEW PR*
@mimilettd @vaidashi @jimszalew

### CODE REVIEW

**Review by vaidashi — COMMENTED** [review]
Looks like it could work, let's go through a live run-through to see how it works.

**Review by jimszalew — DISMISSED** [review]
Looks like solid code. Unfortunately I am not Neo and can't see what the code produces. Thanks for your work on the styling Liam.

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -44,16 +44,36 @@ p{
...
+  border-radius: 5px;
+  background-color: white;
+}
+.item_text {
+  height: 150px;
+}
+.item_title {
+  height: 60px;
+  font-family: 'Caveat Brush', cursive;
+  background: linear-gradient(rgba(0, 0, 0, 0.3), white);
```
Comment (mimizotti) [inline app/assets/stylesheets/application.scss]: @liambarstad I'm going to need you to demo this font just so I can determine that it goes with the overall website.

**Review by mimizotti — APPROVED** [review]
This looks good.

**Pull request merged**